src/utils/tokenizers.py

```diff
@@ -53,8 +53,16 @@ def detokenize(self, tokens):
         '''
         return [self._detokenizer.unescape_xml(t) for t in tokens]
 
+class BertTokenizer(Tokenizer):
+    def __init__(self, bert_model_name):
+        from pytorch_pretrained_bert import BertTokenizer as HFBertTokenizer
```

Comment:
what does HF stand for?

Reply:
Huggingface?

If making a wrapper class, better to name the wrapper something like `BertTokenizerWrapper` and keep the name on the original one.